Sources/Networking/HTTPClient.swift

```diff
@@ -18,23 +18,29 @@ class HTTPClient {
     typealias RequestHeaders = [String: String]
     typealias Completion<Value: HTTPResponseBody> = (HTTPResponse<Value>.Result) -> Void
 
-    private let session: URLSession
     internal let systemInfo: SystemInfo
+    internal let timeoutSeconds: Int
+
+    private let session: URLSession
     private let state: Atomic<State> = .init(.initial)
     private let eTagManager: ETagManager
     private let dnsChecker: DNSCheckerType.Type
 
     init(
         systemInfo: SystemInfo,
         eTagManager: ETagManager,
-        dnsChecker: DNSCheckerType.Type = DNSChecker.self
+        dnsChecker: DNSCheckerType.Type = DNSChecker.self,
+        timeoutSeconds: Int = Configuration.networkTimeoutSecondsDefault
```

Comment:
Should we use `TimeInterval` here too?

Reply:
I was thinking about it, I wanted to get folks' opinions since I don't have a strong one. I lean towards Int only because it's more comfortable as an API for me.

Reply:
✅